frontends/ol-search-ui/src/components/ExpandedLearningResourceDisplay.tsx

```diff
@@ -1,136 +1,119 @@
-import React from "react"
```

Comment:
Major changes in this file:
- removed `setShowResourceDrawer`... The dropdown management is now local to this component.
- Changed the "info" display at bottom of card to use a [`<dl>` (definition list)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) and extracted the logic for formatting the minformation.

    `<dl>`s aren't too common, but I think it is a good fit here.